## GOAL
# This action is used to make the robot perform
# an inspection routine in the current position.

---

## RESULT

# If the routine was completed successfully.
bool completed

---

## FEEDBACK

